test/globalHooks.js

```diff
@@ -4,6 +4,7 @@ before(function (done) {
   const Logger = require('roosevelt-logger')
   const logger = new Logger()
   logger.winstonInstance.silent = true
+  process.env.NODE_ENV = 'test'
```

Comment:
So should we use the `NODE_ENV` environment variable here as there may be a time in the future that we will want to use it in the code for source-configs which then could break things when we set `NODE_ENV` to `test` here.